packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -592,17 +595,35 @@ const conf = (module.exports = convict({
     },
   },
   sentry: {
-    client_errors_dsn: {
-      default: undefined,
-      doc: 'Sentry config for client side errors. If not set, then no errors reported.',
-      env: 'SENTRY_CLIENT_ERRORS_DSN',
+    dsn: {
+      default: '',
+      doc: 'Sentry config for reporting errors. If not set, then no errors reported.',
+      env: 'SENTRY_DSN',
       format: String,
     },
-    server_errors_dsn: {
-      default: undefined,
-      doc: 'Sentry config for Express server-side errors. If not set, then no errors reported.',
-      env: 'SENTRY_SERVER_ERRORS_DSN',
-      format: String,
+    env: {
+      doc: 'Environment name to report to sentry',
+      default: 'local',
+      format: ['local', 'ci', 'dev', 'stage', 'prod'],
+      env: 'SENTRY_ENV',
```

Comment:
Do we need to set `SENTRY_ENV` anywhere in tests for `ci` to get picked up?

We probably also need a PR up in the cloudops repo for these changes, though maybe SRE can do that for us.

Reply:
I think there is a follow up item for dealing with sentry and CI. I'll double check on this. Currently the CI all goes to a special CI project. Ideally we drop this project, and just configure the SENTRY_ENV like you suggest.